src/OpenTelemetry.AutoInstrumentation/Instrumentations/RabbitMq6/Integrations/DefaultBasicConsumerIntegration.cs

```diff
@@ -10,19 +10,20 @@
 namespace OpenTelemetry.AutoInstrumentation.Instrumentations.RabbitMq6.Integrations;
 
 /// <summary>
-/// RabbitMq EventingBasicConsumer.HandleBasicDeliver integration.
+/// RabbitMq DefaultBasicConsumer.HandleBasicDeliver integration.
 /// </summary>
 [InstrumentMethod(
     assemblyName: IntegrationConstants.RabbitMqAssemblyName,
-    typeName: IntegrationConstants.EventingBasicConsumerTypeName,
+    typeName: IntegrationConstants.DefaultBasicConsumerTypeName,
     methodName: IntegrationConstants.HandleBasicDeliverMethodName,
     returnTypeName: ClrNames.Void,
     parameterTypeNames: new[] { ClrNames.String, ClrNames.UInt64, ClrNames.Bool, ClrNames.String, ClrNames.String, IntegrationConstants.BasicPropertiesInterfaceTypeName, $"System.ReadOnlyMemory`1[{ClrNames.Byte}]" },
     minimumVersion: IntegrationConstants.MinSupportedVersion,
     maximumVersion: IntegrationConstants.MaxSupportedVersion,
     integrationName: IntegrationConstants.RabbitMqByteCodeIntegrationName,
-    type: InstrumentationType.Trace)]
-public static class EventingBasicConsumerIntegration
+    type: InstrumentationType.Trace,
+    integrationKind: IntegrationKind.Derived)]
```

Comment:
Similar to my other [comment](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3559/files#r1714300291), can you test that we instrument a consumer of type `RabbitMQ.Client.DefaultBasicConsumer`? I believe we'll need a second `InstrumentMethod` attribute (IntegrationKindDirect) to do so

Reply:
Actually, it seems like the base class doesn't do anything meaningful so this should always be subclasses. Nevermind, we may not need to do a direct instrumentation, sorry for adding confusion